pkg/cmd/container/list.go

```diff
@@ -90,147 +86,69 @@ func filterContainers(ctx context.Context, client *containerd.Client, filters []
 	return upContainers, nil
 }
 
-type containerPrintable struct {
+type ListItem struct {
 	Command   string
-	CreatedAt string
+	CreatedAt time.Time
 	ID        string
 	Image     string
 	Platform  string // nerdctl extension
-	Names     string
+	Names     []string
```

Comment:
?

Reply:
The general principle I'm following for this PR, is to make the pkg layer's data type reflect the intrinsic nature, generic without impacting by the printing preference.

One example to demonstrate this principle is I changed `CreatedAt` from `string` to `time.Time.

For "Names", the naming is a bit confusing because each container can only have one name.

But I looked into the Docker API [spec](https://docs.docker.com/engine/api/v1.43/#tag/Container/operation/ContainerList) and found Names is Array in `/containers/json` response. That made me think it is not just confusing naming but there may be some design consideration behind the scene to keep the flexibility of supporting multiple names in the future.

So I tend to think []string can reflect more intrinsic nature. However, the evidences are not that strong so I don't have strong preference. Happy to ignore the naming and make it completely `string` if you prefer it.